AUR Service Tests Failing

:clock11: **When did the problem start?**
~12 days ago
<!-- Indicate when the problem started -->

:camera: **Live badge**
Any valid package will work, issue is with the package several of the service tests are using
<!-- Provide a link to the live badge in plain text and markdown.  -->

![]( url goes here )

:wrench: **Is the live badge working?**
Yes, see above
<!-- Indicate whether or not the live badge is working. -->

:link: **CircleCI link**

<!-- Provide a link to the failing test in CircleCI. -->
https://circleci.com/gh/badges/daily-tests/323
:beetle: **Stack trace**

```
AssertionError: message mismatch: expected 'package not found' to equal 'GPL'
    at Function._expectField (core/service-test-runner/icedfrisby-shields.js:48:51)
    at IcedFrisbyNock.afterJSON.json (core/service-test-runner/icedfrisby-shields.js:38:26)
    at IcedFrisbyNock.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:924:10)
    at invokeNextHook (node_modules/icedfrisby/lib/icedfrisby.js:973:24)
    at Promise (node_modules/icedfrisby/lib/icedfrisby.js:987:7)
    at new Promise (<anonymous>)
    at IcedFrisbyNock._runHooks (node_modules/icedfrisby/lib/icedfrisby.js:946:12)
    at IcedFrisbyNock.run (node_modules/icedfrisby/lib/icedfrisby.js:1238:20)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:229:7)
```

:bulb: **Possible solution**
Just need to swap the target package in a few of the tests
<!--- Optional: only if you have suggestions on a fix/reason for the bug -->

<!-- Love Shields? Please consider donating $10 to sustain our activities:
👉  https://opencollective.com/shields -->

--- a/services/aur/aur.service.js
+++ b/services/aur/aur.service.js
@@ -61,8 +61,8 @@ class AurLicense extends BaseAurService {
     return [
       {
         title: 'AUR license',
-        namedParams: { packageName: 'yaourt' },
-        staticPreview: this.render({ license: 'GPL' }),
+        namedParams: { packageName: 'pac' },
+        staticPreview: this.render({ license: 'MIT' }),
       },
     ]
   }
@@ -96,8 +96,8 @@ class AurVotes extends BaseAurService {
     return [
       {
         title: 'AUR votes',
-        namedParams: { packageName: 'yaourt' },
-        staticPreview: this.render({ votes: '3029' }),
+        namedParams: { packageName: 'dropbox' },
+        staticPreview: this.render({ votes: '2280' }),
       },
     ]
   }
@@ -135,8 +135,8 @@ class AurVersion extends BaseAurService {
     return [
       {
         title: 'AUR version',
-        namedParams: { packageName: 'yaourt' },
-        staticPreview: this.render({ version: 'v1.9-1', outOfDate: null }),
+        namedParams: { packageName: 'visual-studio-code-bin' },
+        staticPreview: this.render({ version: '1.34.0-2', outOfDate: null }),
       },
     ]
   }

--- a/services/aur/aur.tester.js
+++ b/services/aur/aur.tester.js
@@ -14,7 +14,7 @@ const t = (module.exports = new ServiceTester({
 // version tests
 
 t.create('version (valid)')
-  .get('/version/yaourt.json')
+  .get('/version/dropbox.json')
   .expectBadge({
     label: 'aur',
     message: isVPlusDottedVersionNClausesWithOptionalSuffix,
@@ -36,7 +36,7 @@ t.create('version (not found)')
 // votes tests
 
 t.create('votes (valid)')
-  .get('/votes/yaourt.json')
+  .get('/votes/discord.json')
   .expectBadge({
     label: 'votes',
     message: isMetric,
@@ -49,8 +49,8 @@ t.create('votes (not found)')
 // license tests
 
 t.create('license (valid)')
-  .get('/license/yaourt.json')
-  .expectBadge({ label: 'license', message: 'GPL' })
+  .get('/license/pac.json')
+  .expectBadge({ label: 'license', message: 'MIT' })
 
 t.create('license (not found)')
   .get('/license/not-a-package.json')

